### PR TITLE
Use github actions for example project

### PR DESCRIPTION
**Pull request by nllong**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        run: |
+          python -c "import sys; print(sys.version)"
+          docker --version
+          docker-compose --version
+      -
+        name: Install dependencies
+        env:
+          MODELICAPATH: /home/runner/work/modelica-buildings
+        run: |
+          git clone --single-branch --branch issue2204_gmt_mbl https://github.com/lbl-srg/modelica-buildings.git $MODELICAPATH
```
Comment (nllong) [inline .github/workflows/ci.yml:32]: This contains the update to the MBL branch for the new workflow.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      - uses: actions/checkout@v2
+      -
+        name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      -
+        name: Display system info
+        run: |
+          python -c "import sys; print(sys.version)"
```
Comment (vtnate) [inline .github/workflows/ci.yml:24]: Is there a benefit to replacing this whole line with `python --version`? It shows less information than the way you've got it currently. Is overhead a concern?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+on:
+  pull_request:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # 3.8, 3.9 do not work at the moment.
```
Comment (vtnate) [inline .github/workflows/ci.yml:12]: Python 3.8 & 3.9 don't work? What's up with that? I'm using 3.8.5 on my machine, should I be using 3.7?

**Review by vtnate — APPROVED** [review]
I ran a trivial PR against this branch to test, worked as expected. Woohoo!

I'll delete this test PR at lunch time today. Here it is if you care to check it out.
https://github.com/urbanopt/geojson-modelica-translator-examples/pull/6

File: .github/workflows/ci.yml (hunk 3 above)
Comment (nllong) [reply .github/workflows/ci.yml:12]: I dunno. need to investigage. If you are using 3.8 successfully with GMT then maybe all is good.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (nllong) [reply .github/workflows/ci.yml:24]: unsure... it is what is in the GitHub workflow template.

**Pull request merged**